lib/normalizers/v1.js

```diff
@@ -225,27 +229,39 @@ _.assign(Builders.prototype, {
         collectionId && !noDefaults && (requestV1.collectionId = collectionId);
 
         // normalized v1 requests should not have falsy helperAttributes or currentHelper
-        if (_.has(requestV1, 'currentHelper') && !requestV1.currentHelper) {
-            delete requestV1.currentHelper;
+        if (_.has(requestV1, 'currentHelper')) {
+            (requestV1.currentHelper === 'normal') && (requestV1.currentHelper = null);
+
+            if (!requestV1.currentHelper) {
+                (requestV1.currentHelper !== null) && (requestV1.currentHelper = null);
```

Comment:
`requestV1.currentHelper = null` The only other case would be `requestV1.currentHelper === null`. It doesn't warrent to be handled explicilty.